server/handlers_test.go

```diff
@@ -3832,7 +3832,7 @@ func TestHTTPServer_GroupsEndpoint(t *testing.T) {
 			StatusCode: http.StatusOK,
 			Body:       expectedBody,
 		})
-	}, testTimeout)
+	}, 30*time.Second)
```

Comment:
why do you use `30s` instead of the `testTimeout`?

Reply:
because these two test cases are very unstable and sometimes it takes more than 10s. I found that 30s was safe enough for this while not being too long in case it fails for a real reason.